### DISCUSSION TITLE
Looking for 2 boards v54 rev 1.39 or newer (AC variant)

### DISCUSSION THREAD

**Discussion by herrfrei** [nliaudat/esp32_8ch_motor_shield#36] [Buy or Sell]
- with assembled SMT parts, RJ11 / screw terminal
- without ESP
- delivery to Germany 
 
Any offers?

**Comment by WMP** [comment]
I can send you from Poland v57 rev 1.48 with custom box 
![image](https://user-images.githubusercontent.com/45381/232234992-5e309acf-dd08-4f12-8a27-abd10c409936.png)

**Reply by herrfrei** [reply]
Looks fine. What's the price (with and without the box) and shipping costs?

**Reply by WMP** [reply]
Shipping 10-15 EUR
Assembled PCB 27eur
I will know the price of the case in a week

**Reply by nliaudat** [reply]
Can you share you box ? It looks fine !

**Reply by WMP** [reply]
No, you have to wait for the final version and accept my pr :)

**Reply by herrfrei** [reply]
> Shipping 10-15 EUR Assembled PCB 27eur I will know the price of the case in a week

The case is not important for me (I think I need to design a special one for my needs). So 2 PCBs plus shipping is enough for me.

**Reply by WMP** [reply]
I send you mail to address from GitHub profile

**Comment by heinemannj** [comment]
> Shipping 10-15 EUR
> Assembled PCB 27eur
> I will know the price of the case in a week

Looking for 1x assembled PCB v57 rev 1.48 or newer
- with assembled SMT parts, RJ11 / screw terminal
- with custom box
- without ESP
- delivery to Germany

Any offers?

**Reply by WMP** [reply]
I have few 1.48 and if you want i can print you custom box. Please send me mail.

**Comment by MrAges** [comment]
Hello,

I´m looking for 3x assembled PCB v57 rev 1.48

- without ESP
- delivery to Austria

Are there any left?
